### ISSUE TITLE
Support for Java Spring boot/security

### ISSUE THREAD

**Issue by aquilares** [IdentityServer/IdentityServer4#2890]
I was wondering: does identity server also integrate with e.g. java spring boot applications?
Including the webhooks/redirects for login, password reset, registration, ...?

Or is this solution only compatible with .net (core) applications?
I'm looking for an alternative to Keycloak and was wondering if identityserver could cover my needs :)

**Comment by leastprivilege** [comment]
IdentityServer is an OpenID Connect/OAuth framework. You can integrate anything that is compatible with these protocols.

**Comment by leastprivilege** [comment]
That said - anything that is outside of these protocols (like login UI or user management) is not part of IdentityServer - so you have to implement it yourself and thus can make it compatible with whatever you want.

**Comment by aquilares** [comment]
Thanks for the reply @leastprivilege 

When reading this: [https://identityserver4.readthedocs.io/en/latest/quickstarts/3_interactive_login.html](url) => am I wrong in assuming that, if I e.g. create a separate .net core project with identity server, drop in these UI-components and run it in a separate docker container, that I could get identity server to offer these user management/login GUI components? 

So basically redirecting back and forward during the authentication flow between my own app and identitserver?

**Comment by leastprivilege** [comment]
Yes of course.
